pytorch.yml

```
@@ -16,3 +14,6 @@ dependencies:
   - torchvision
   - pynvml
   - matplotlib
+  - pip
+  - pip:
+    - optuna-integration
```

Why `pip`? It looks like this is available on conda-forge https://anaconda.org/conda-forge/optuna-integration

---

Ah, I tried briefly and mamba didn't find it.  It's worth checking again though.

---

I tried locally with these changes and mamba resolved the environment 

```suggestion
  - optuna-integration
```